roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -25,7 +25,8 @@
     signer_serial: "{{ openshift_master_config_dir }}/ca.serial.txt"
     hostnames:
     - "{{ docker_registry_service_ip.results.clusterip }}"
-    - docker-registry.default.svc.cluster.local
+    - docker-registry.default.svc
+    - "docker-registry.default.svc.{{ openshift.common.dns_domain }}"
```

Comment:
@sdodson,

I'm not sure it is static as we are creating the service here: 

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_hosted/tasks/registry/registry.yml#L56-L67

The variable is defined in registry.yml: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_hosted/tasks/registry/registry.yml#L36

I'd prefer if we just used it since it will future proof us.